src/Symfony/Component/Console/Question/ChoiceQuestion.php

```diff
@@ -32,6 +32,10 @@ class ChoiceQuestion extends Question
      */
     public function __construct($question, array $choices, $default = null)
     {
+        if (!$choices) {
+            throw new \LogicException('Choice question must have at least 1 choice available.');
```

Comment:
not sure if we really care :) imo. the argument is valid (an array), but it's domain is invalid (empty array). So `DomainException` actually (imo.) but i thought `LogicException` is a bit more common in SF :)

Im fine either way.. as long as it's extends from logic i dont mind :)